frontend/src/pages/Notices .jsx

```
@@ -0,0 +1,116 @@
+        )}
+
+        {/* Display Notices */}
+        {!loading && !error && (
+          <div className="space-y-6">
+            {notices.length > 0 ? (
+              notices.map((notice) => (
+                  <button
+                      key={notice.id}
+                      onClick={() => handleNoticeClick(notice.id)}
+                      className="w-full text-left p-6 transition-shadow duration-300 ease-in-out bg-white border border-gray-200 rounded-lg shadow-md cursor-pointer hover:shadow-lg"
+                  >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)